test/fail_compilation/fail11714.d

```diff
@@ -0,0 +1,21 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail11714.d(14): Error: variable fail11714.c is a thread-local class and cannot have a static initializer. Use `static this()` to initialize instead.
+fail_compilation/fail11714.d(21): Error: variable fail11714.s is a thread-local pointer to struct and cannot have a static initializer. Use `static this()` to initialize instead.
```

Comment:
Would be nice if `fail11714.c` and `fail11714.s` had tick marks around them.

Reply:
That would be a change for another pr. 😉

Reply:
Also, every source file in the testsuite would need updating if you make that change.